source/common/protobuf/utility.h

```diff
@@ -329,6 +329,26 @@ class MessageUtil {
    */
   static void unpackTo(const ProtobufWkt::Any& any_message, Protobuf::Message& message);
 
+  /**
+   * Convert from google.protobuf.Any to bytes as std::string
+   * @param message source google.protobuf.Any message.
+   *
+   * @return std::string consists of bytes in the input message.
+   */
+  static std::string anyToBytes(const ProtobufWkt::Any& any) {
```

Comment:
Would it be possible to add some basic small tests that verify that this method works as intended?

Reply:
Thanks, will do!

Reply:
see https://github.com/envoyproxy/envoy/pull/15326